core/src/main/java/io/questdb/std/Encoding.java

```diff
@@ -0,0 +1,143 @@
+package io.questdb.std;
+
+import io.questdb.cairo.CairoException;
+import io.questdb.std.str.CharSink;
+
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+
+public final class Encoding {
```

Comment:
would it be possible to keep this in `Chars.` There is refactoring moving `encoding` to another class, which makes it hard to see changes. Also `Encoding` is too generic, basically as generic as `Chars`.

Generally there is nothing wrong with having `Encoding`, but I am trying to address utility class prolifiration, which makes it harder to locate utility methods

Reply:
done